src/sx-design/src/Menu/__tests__/Menu.test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @flow
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+
+import { render, fireEvent } from '../../test-utils';
+import Menu from '../Menu';
+
+it('renders the menu without any issues', () => {
+  const { getByRole, getByText, getByTestId, queryByText, queryByTestId } = render(
+    <Menu>
+      <Menu.Item onClick={jest.fn()}>One</Menu.Item>
+      <Menu.Item onClick={jest.fn()}>Two</Menu.Item>
+      <Menu.Item tint="default" onClick={jest.fn()}>
+        Three
+      </Menu.Item>
+      <Menu.ItemDivider data-testid="MenuItemDivider" />
+      <Menu.Item tint="error" onClick={jest.fn()}>
+        Delete
+      </Menu.Item>
+    </Menu>,
+  );
+
+  // Everything is hidden by default (not in the DOM):
+  expect(queryByText('One')).toBeNull();
+  expect(queryByText('Two')).toBeNull();
+  expect(queryByText('Three')).toBeNull();
+  expect(queryByTestId('MenuItemDivider')).toBeNull();
+  expect(queryByText('Delete')).toBeNull();
+
+  // We have to first open the menu:
+  fireEvent.click(getByRole('button'));
```

Comment:
Probably not the most important, but there is a package that is better for user events https://testing-library.com/docs/ecosystem-user-event/

Reply:
I will have a look at it thanks! Since I am not familiar with this package: do you know how does it differ from the `fireEvent`? In other words: why does `fireEvent` even exist if it's better to use something else? 🤔

Reply:
I think userEvent also triggers hover, mouseOver etc before clicking the button, while the fireEvent does not. I think they were planning on merging this into the main package at some point. 

I'll see if I can find the article for you once I have time.

Reply:
I am already preparing a PR to change the whole package from `fireEvent` to `userEvent`! 👍